tests/test_dataset.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import unittest
+
+sys.path.append("./src")
+
+
+class DatasetTestCase(unittest.TestCase):
+    def test_graph(self):
+        from modelset import load
+        dataset = load(modeltype='ecore', selected_analysis=['stats'])
```

Comment:
I'm not sure if this is working as expected. This is trying to use the dataset installed in `~/.modelset` but I'm not sure if this is the correct approach. I think that the approach of assuming that there is a sibling project `modelset-dataset` looks more stable to me.

Reply:
I documented the test.